packages/utilities/psammead-locales/moment/sw.test.js

```diff
@@ -141,14 +141,18 @@ test('from', function () {
 
 test('suffix', function () {
   assert.equal(moment(30000).from(0), 'hivi punde baadaye',  'prefix');
-  assert.equal(moment(0).from(30000), 'tokea hivi punde', 'suffix');
+  assert.equal(moment(0).from(30000), 'hivi punde zilizopita', 'suffix');
```

Comment:
`hivi punde zilizopita` doesn't sound grammaticality correct. I think just "`hivi punde` should suffice.

Cc @Bopchy

Reply:
Yes, it doesn't sound quite right. `hivi punde` should be enough

Reply:
I think `sekunde chache zilizopita` would be a better translation instead.  It means `a few seconds ago`.

Reply:
Updated :)